epiviz/epiviz-mbrowser
# Sprint 2: mBrowser filter, column config, project summary components

This PR includes:
- populating the filter component (does NOT include functionality)
- adding the project summary card
- adjust styling and bugs from previous sprint

To test:
- Make sure you have a local copy of this repository 
`git clone https://github.com/epiviz/epiviz-mbrowser.git`
- checkout this branch
`git checkout sprint2-filter`
- install dependencies
`npm install`
- run the app
`npm start`
